examples/nordic_example.py

```diff
@@ -0,0 +1,95 @@
+"""
+===========================================================
+Apply NORDIC to complex-valued fMRI data with noise volumes
+===========================================================
+
+This example shows how to apply NORDIC to complex-valued fMRI data with noise
+volumes.
+"""
+
+###############################################################################
+# Load Data
+# -----------------------------------------------------------------------------
+
+import nibabel as nb
+import numpy as np
+
+magnitude_file = 'data/sub-01_task-rest_part-mag_bold.nii.gz'
+phase_file = 'data/sub-01_task-rest_part-phase_bold.nii.gz'
+magnitude_noise_file = 'data/sub-01_task-rest_part-mag_noRF.nii.gz'
+phase_noise_file = 'data/sub-01_task-rest_part-phase_noRF.nii.gz'
+
+magnitude_img = nb.load(magnitude_file)
+phase_img = nb.load(phase_file)
+magnitude_noise_img = nb.load(magnitude_noise_file)
+phase_noise_img = nb.load(phase_noise_file)
+
+magnitude_data = magnitude_img.get_fdata()
+phase_data = phase_img.get_fdata()
+magnitude_noise_data = magnitude_noise_img.get_fdata()
+phase_noise_data = phase_noise_img.get_fdata()
+
+n_noise_volumes = magnitude_noise_data.shape[3]
+
+magnitude_data = np.concatenate((magnitude_data, magnitude_noise_data), axis=3)
+phase_data = np.concatenate((phase_data, phase_noise_data), axis=3)
+
+# Rescale phase data to [-pi, pi]
+phase_range = np.max(phase_data)
+phase_range_min = np.min(phase_data)
+range_norm = phase_range - phase_range_min
+range_center = (phase_range + phase_range_min) / range_norm * 1 / 2
+phase_data = (phase_data / range_norm - range_center) * 2 * np.pi
+
+# Combine magnitude and phase into complex-valued data
+complex_data = magnitude_data * np.exp(1j * phase_data)
+
+# TODO: Filter complex data and calculate g-factor map
```

Comment:
@paquiteau would adding these steps be useful for an example, or should they go into a function/class within patch-denoising?

Reply:
Well, this is already done in the `load_complex_nifti` function, which is what people would use in practice. I don't think it's needed to refactor that.

Reply:
Are you referring to combining the complex-valued data or the phase filtering and g-factor map creation? `load_complex_nifti` doesn't do the latter. Also, it doesn't rescale the phase data to -pi to pi, right? So this example definitely needs to separately rescale the data.